community/cypher/cypher-compiler-3.1/src/main/scala/org/neo4j/cypher/internal/compiler/v3_1/mutation/CreateRelationship.scala

```diff
@@ -68,7 +68,7 @@ extends UpdateAction
   def localEffects(symbols: SymbolTable) = props.values.foldLeft(Effects(CreatesRelationship(typ)))(_ ++ _.effects(symbols))
 
   override def children =
-    props.values.toSeq ++ Seq(from.node, to.node) ++
+    props.values.toIndexedSeq ++ Seq(from.node, to.node) ++
```

Comment:
Will this make `children` an `IndexedSeq`? Perhaps the literal `Seq` that gets appended needs to be a `Vector`?